lib/config.js

```diff
@@ -41,4 +41,41 @@ export default {
     maximum: 1.0,
     order: 5,
   },
+
+  projectDefaults: {
+    type: "object",
+    title: "Default Project Settings",
+    properties: {
+      outputPath: {
+        title: "Output Path",
+        description: "The path relative to a project root where the compilation \
+                      output will be saved.",
+        type: "string",
+        default: ".latex",
+      },
+
+      postCompileAction: {
+        title: "Post Compile Action",
+        description: "Action to be performed after the compilation. Note that, if \
+                      `link` or `copy` is chosen, any file at `Post Compile Action \
+                      Path` whose name clashes with `'texroot'.pdf` will be overwritten. \
+                      Invoke the `edit` command to chek a project `texroot`.",
+        type: "string",
+        default: "none",
```

Comment:
I would like to have the default be what the current behavior is so that current users aren't forced to change any settings. Would you mind setting this to "link?"